src/ctia/entity/feedback.clj

```diff
@@ -52,7 +53,8 @@
      JudgementFieldsParam
      {(s/optional-key :sort_by) feedback-sort-fields})))
 
-(def feedback-by-entity-route
+(s/defn feedback-by-entity-route [{{:keys [read-store]} :StoreService
+                                   :as _services_} :- APIHandlerServices]
```

Comment:
```suggestion
(s/defn feedback-by-entity-route [{{:keys [read-store]} :StoreService} :- APIHandlerServices]
```

Reply:
Leaving as it makes my life a bit easier.